client/api/src/backend.rs

```diff
@@ -195,8 +195,9 @@ pub trait BlockImportOperation<Block: BlockT> {
 	fn mark_finalized(
 		&mut self,
 		id: BlockId<Block>,
-		justification: Option<Justification>,
+		justifications: Option<Justifications>,
```

Comment:
I think this should be kept as it was. I don't see a situation where you need to finalize an existing block and you have multiple justifications at hand.